src/infra/marketplace/marketplaceQueryRepo.go

```diff
@@ -355,6 +355,33 @@ func (repo *MarketplaceQueryRepo) parseCatalogItemCmdSteps(
 	return itemCmdSteps, nil
 }
 
+func (repo *MarketplaceQueryRepo) parseCatalogItemFilesToDelete(
+	catalogItemFilesToDelete interface{},
+) (itemFilesToDelete []valueObject.UnixFileName, err error) {
+	if catalogItemFilesToDelete == nil {
+		return itemFilesToDelete, nil
+	}
+
+	rawItemFilesToDelete, assertOk := catalogItemFilesToDelete.([]interface{})
```

Comment:
Can't you cast this as []string{} directly?

Reply:
In this case, no, because I am passing a property mapped as `interface{}` from the factory. When it arrives here, it expects an `[]interface{}`. If I try to use type assertion with `[]string`, it gives an error.